host.yaml

```diff
@@ -0,0 +1,92 @@
+ref: "openshift/3.10/x86_64/os"
+repos:
+  - atomic-centos-continuous
+  - rhcos-continuous
+  - openshift
+  - rhel-7.5-server
+  - rhel-7.5-server-optional
+  - rhel-7.5-server-extras
+  - rhel-7.5-atomic
+install_langs:
+ - en_US
+documentation: false
+initramfs-args:
+  - "--no-hostonly"
+  - "--add"
+  - "iscsi"
+automatic_version_prefix: "3.10-7.1803"
```

Comment:
Minor: let's stick with `3.10-7.5.0`? (I'm guessing this was just forgotten during the transition from inheriting JSON files to a single YAML).

Reply:
I changed to use `7.5` since...the `.0` there feels like it's claiming something stronger than we're really saying.